plugins/graphql-kotlin-maven-plugin/src/main/kotlin/com/expediagroup/graphql/plugin/maven/GenerateClientAbstractMojo.kt

```diff
@@ -99,17 +99,23 @@ abstract class GenerateClientAbstractMojo : AbstractMojo() {
 
     override fun execute() {
         log.debug("generating GraphQL client")
-        val graphQLSchemaFile = schemaFile ?: File(project.build.directory, "schema.graphql")
-        validateGraphQLSchemaExists(graphQLSchemaFile)
+
+        val schemaPath =
```

Comment:
you can rewrite this as 
```suggestion
        val schemaPath = schemaFile ?: File(project.build.directory, "schema.graphql").path
```
which avoids the `!!`